Dockerfile

```diff
@@ -0,0 +1,37 @@
+# sources:
+# - https://github.com/jupyter/docker-stacks
+# - https://github.com/TomographicImaging/CIL#installation-of-cil
+# consumers:
+# - https://github.com/stfc/cloud-docker-images
+# TODO: use `ubuntu-22.04` after `python=3.11` is supported, vis.
+# https://github.com/TomographicImaging/CIL/issues/1490
+FROM jupyter/tensorflow-notebook:ubuntu-20.04
+LABEL org.opencontainers.image.source=https://github.com/TomographicImaging/CIL
+LABEL org.opencontainers.image.description="Core Imaging Library"
+LABEL org.opencontainers.image.licenses="Apache-2.0"
+
+# CUDA-specific packages
+ARG CIL_EXTRA_PACKAGES=astra-toolbox tigre
+# build & runtime dependencies
+# TODO: sync with scripts/requirements-test.yml
+# TODO: sync with scripts/create_local_env_for_cil_development.sh
```

Comment:
I agree with this. Can you not call

`mamba env update -f scripts/requirements-test.yml`?